oracleAQ/java/src/main/java/com/aq/java/OracleAQController.java

```diff
@@ -0,0 +1,64 @@
+package com.aq.java;
```

Comment:
naming the directory "java" makes things confusing/ambiguous with the "main/java" src location. please rename it to "aq-java" or some such